meta-client/src/main/java/org/apache/helix/metaclient/impl/zk/ZkMetaClient.java

```diff
@@ -155,12 +209,15 @@ public void asyncTransaction(Iterable iterable, AsyncCallback.TransactionCallbac
 
   @Override
   public boolean connect() {
-    return false;
+    // TODO: This is a tempp impl for test only. no proper event handling and error handling.
+    _zkClient.connect(Integer.MAX_VALUE, _zkClient);
+    return true;
```

Comment:
For my own understanding, does this work with the current test?
( I remember during my own test, it fails if the zkclient has already connected, which is the case in native zkclient as it's connected during initialization.)

Reply:
Thanks for bring this up. this connect is not called in test as of now. I plan to change it after your PR that includes constructor change is added.